examples/config-gc.json

```diff
@@ -3,10 +3,47 @@
     "storage": {
         "rootDirectory": "/tmp/zot",
         "gc": true,
-        "gcReferrers": true,
         "gcDelay": "2h",
         "untaggedImageRetentionDelay": "4h",
-        "gcInterval": "1h"
+        "gcInterval": "1h",
+        "retention": {
+            "dryRun": false,
+            "policies": [
+                {
+                    "repoNames": ["infra/*", "prod/*"],
+                    "deleteReferrers": false,
+                    "deleteUntagged": true,
+                    "tagsRetention": [{
+                        "names": ["v2.*", ".*-prod"]
+                    },
+                    {
+                        "names": ["v3.*", ".*-prod"],
+                        "pulledWithinLastNrDays": 7
```

Comment:
"pulledWithin": "7d"?

Reply:
https://pkg.go.dev/time#ParseDuration

the biggest unit is h(our) 
it gets hard to express weeks/months, I changed it, but I can revert it back